hack/update-deps.sh

```diff
@@ -28,9 +28,6 @@ dep ensure
 rm -rf $(find vendor/ -name 'OWNERS')
 rm -rf $(find vendor/ -name '*_test.go')
 
-# Keep the only dir in knative/test-infra we're interested in
```

Comment:
Please don't remove this code, update it to keep only the relevant code from test-infra instead.